playbooks/bootstrap.yml

```diff
@@ -15,6 +15,9 @@
   any_errors_fatal: "{{ edpm_any_errors_fatal | default(true) }}"
   max_fail_percentage: "{{ edpm_max_fail_percentage | default(0) }}"
   tasks:
+    - name: Gather ansible_local facts
+      ansible.builtin.setup:
+        filter: ansible_local
```

Comment:
Yeah, we should realistically do this at the start of each of our service playbooks:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/874